# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libspiff/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libspiff-1.0.0rc2/bindings/c/spiff_c.cpp.vanilla	2008-09-02 10:20:02.000000000 +0200
+++ libspiff-1.0.0rc2/bindings/c/spiff_c.cpp	2008-09-02 10:20:21.000000000 +0200
@@ -53,6 +53,8 @@
 #include <spiff/SpiffTrackWriter.h>
 #include <spiff/SpiffWriter.h>
 
+#include <string.h>
+
 extern "C" {
 #include "spiff_c.h"
 }
